modules/ding_place2book/ding_place2book.module

```diff
@@ -531,7 +538,9 @@ function ding_place2book_get_event_state($event_maker_id, $event_id) {
         break;
     }
 
-    $output = "<span class='place2book button {$class}'>{$data}</span>";
+    if ($status !== 'passive') {
```

Comment:
I would define empty also, as there might be cases we didn’t cover.

Reply:
👍 I will try to refactor the logic.

Reply:
Why not just only show this if status is set? I mean what is the reason to set the status to passive on L467

Reply:
I removed the passive check and changed to empty check in the end